contracts/src/external/HyperdriveTarget4.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { IHyperdrive } from "../interfaces/IHyperdrive.sol";
+import { HyperdriveAdmin } from "../internal/HyperdriveAdmin.sol";
+import { HyperdriveCheckpoint } from "../internal/HyperdriveCheckpoint.sol";
+import { HyperdriveLong } from "../internal/HyperdriveLong.sol";
+import { HyperdriveLP } from "../internal/HyperdriveLP.sol";
+import { HyperdriveMultiToken } from "../internal/HyperdriveMultiToken.sol";
+import { HyperdriveShort } from "../internal/HyperdriveShort.sol";
+import { HyperdriveStorage } from "../internal/HyperdriveStorage.sol";
+
+/// @author DELV
+/// @title HyperdriveTarget4
+/// @notice Hyperdrive's target 4 logic contract.
```

Comment:
I never understood what your pattern for target4 vs target 4 was

Reply:
I just keep it consistent. I think this might be the only place we use `target 4`. I can change it if it bothers you.